cmd/workflow-controller/main.go

```diff
@@ -58,6 +58,7 @@ func NewRootCommand() *cobra.Command {
 		workflowTTLWorkers      int    // --workflow-ttl-workers
 		podCleanupWorkers       int    // --pod-cleanup-workers
 		cronWorkflowWorkers     int    // --cron-workflow-workers
+		archivedWorkflowWorkers int    // --archived-workflow-workers
```

Comment:
Mmm adding a new flag certainly sounds like a feature to me, not a fix

Reply:
I change the commit message to a feature

Reply:
The PR title is the biggest part, so I updated that as well. Thanks